data/campaigns/Heir_To_The_Throne/scenarios/19c_Cliffs_of_Thoria.cfg

```diff
@@ -767,24 +773,34 @@ Soooo... It is you who sent your subordinates to attack us. Now when we’ve des
         [/message]
         [message]
             speaker="Li'sar"
-            message= _ "Whatever... I still think we should make an effort to rescue him from the unfortunate position he is in. If you help him, I’ll do my best to convince him that you are not his enemies but his friends..."
-        [/message]
-        [message]
-            speaker=Konrad
-            message= _ "Our main goal remains to survive ourselves, but if we can spare the troops, we might try to rescue the sergeant stranded on the cliff. However, I make no promises."
+            message= _ "Be that as it may, I still think we should make an effort to rescue him from the unfortunate position he is in. If you help him, I’ll do my best to convince him that you are not his enemies but his friends."
         [/message]
         [message]
             speaker=Kalenz
-            message= _ "Fair enough!"
+            message= _ "Very well. If you convince him not to raise his sword against us, the Lintair elves shall grant him safe passage."
         [/message]
         [message]
-            speaker="Li'sar"
-            message= _ "Fair enough."
+            speaker=Konrad
+            message= _ "Our main goal remains to survive ourselves, but if we can spare the troops, we might try to rescue the sergeant stranded on the cliff. However, I make no promises."
         [/message]
         [message]
-            speaker=Delfador
-            message= _ "Fair enough..."
-        [/message]
+            speaker="Li'sar"
+            message= _ "That I do understand, Konrad. I am no stranger to the burden of command."
+        [/message]
+        {IF_VAR waterfall_sighted boolean_equals yes (
+            [then]
+                [message]
+                    speaker=Konrad
+                    message= _ "I’m glad we agree, Princess. Now, onward! We must reach the cave."
+                [/message]
+            [/then]
+            [else]
+                [message]
+                    speaker=Konrad
+                    message= _ "I’m glad we agree, Princess. Now, onward! We must continue downstream, past these drakes."
```

Comment:
```suggestion
                    message= _ "I’m glad we agree, Li'sar. Now, onward! We must continue downstream, past these drakes."
```

Reply:
Same. Also, there's a bunch of uses of `Prince` as a direct address:

- scenarios/01_The_Elves_Besieged.cfg:376:            message= _ "Prince... you must keep fighting! Nooooooo!" (speaker=Delfador)
- scenarios/09_The_Valley_of_Death.cfg:424:                            message= _ "He died fighting for you, Prince, just as any of us would be willing to die for you." (speaker=Kalenz)
- scenarios/14_Plunging_Into_the_Darkness.cfg:532:                    message= _ "I am sorry, Prince Konrad. The young gryphons are breaking their ropes. They are simply getting too large and too restless to restrain anymore!" (speaker=Gryphon Tender)
- scenarios/15_The_Lost_General.cfg:634:                    message= _ "And fine troops they are, Prince, but no dwarf can match a guardsman’s resolute defense. Please accept the service o’ my clan’s defenders, to protect you and your quest." (speaker=Geldar)
- scenarios/17_Scepter_of_Fire.cfg:668:                        message= _ "Prince Konrad, that din was made by no living thing. It was the sound of the very earth that’s boiling around us, of the fire and lava that are breaching this cave even now. We must tread with caution, for the very ground we stand on may turn against us." (speaker=mage)
- scenarios/18_A_Choice_Must_Be_Made.cfg:244:            message= _ "We should indeed seek refuge there, my lord, but not by going east along the river. Its name means ‘The River of Bones’. Great and evil creatures lurk along its banks, its waters are not fit to drink, and it runs over the Cliffs of Thoria. It has been many centuries since any man or elf has passed over the Cliffs and survived. No, Prince, we must choose another way." (speaker=Kalenz; note "my lord" also refers to Konrad)
- scenarios/18_A_Choice_Must_Be_Made.cfg:288:                    message= _ "Prince Konrad, you need not avoid the Arkan-thoria. We merfolk can lead you downstream; with our vanguard, your company will vanquish any foe, and reach the home of the North Elves much sooner than by either of the land routes. I can not promise a perilless journey, but it is an option for you to consider, my lord." (speaker=mer)
- scenarios/18_A_Choice_Must_Be_Made.cfg:365:            message= _ "Prince Konrad, are you sure you want to do this?" (speaker=mer)
- scenarios/19c_Cliffs_of_Thoria.cfg:349:            message= _ "Eh... well of course, sir drake. Let me introduce myself — I am Prince Konrad, leader of the group and heir to the throne of Wesnoth." (speaker=Konrad)
- scenarios/19c_Cliffs_of_Thoria.cfg:369:            message= _ "The Prince speaks the truth." (speaker=Kalenz)

What do you think should be done about them? I added the speakers in parentheses.

Reply:
We can probably fix them in another PR, or I'll probably just make some commits to write over them later if/when I remember to (or you can if you like). Or you could even do them here if you wanted to, up to you.

Reply:
#4392